tests/logic/email_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import mock
+import unittest
+
+import logic.email
+
+
+class EmailTest(unittest.TestCase):
+    """We really just want to test that configuration is honored here."""
+
+    sender = 'test@example.com'
```

Comment:
Right now `get_name_and_email` isn't executed at all, because I'm mocking out the email backends -- this really only tests that `send_email` chooses the correct email backend. The `send_sendgrid_email()` function requires the `sendgrid` client library to be installed, which it may not be if the feature is disabled.

My rationale was that extra requirements can be really annoying (pinning them and subsequently updating them, and dealing with breakages, etc), so I didn't want to force a dependency on `sendgrid` for Yelp and others that don't use it. If the extra dependency isn't a big deal I can see about getting rid of that.

In either case, I may be able test a level deeper by trying to mock the underlying email APIs, which would result in testing `get_name_and_email` as well.